src/libsyncengine/requests/exclusiontemplatecache.cpp

```diff
@@ -145,26 +145,26 @@ ExitCode ExclusionTemplateCache::update(bool def, const std::vector<ExclusionTem
     return ExitCodeOk;
 }
 
-bool ExclusionTemplateCache::checkIfIsExcluded(const SyncPath &basePath, const SyncPath &relativePath, bool &isWarning,
```

Comment:
The original name is certainly both awful and painful to type. However, it remains closer to what the function actually does: it checks, among other things, whether a file is excluded; the `isExcluded` containing the specific exclusion information.

The issue with your proposal is that the caller may think that `isExcluded` returns the exclusion status of the file under scrutiny. It actually returns of the function call: `true` if the function was able to retrieve the desired information, false if some unexpected error was encountered.

Reply:
Mmmh yes you are right. It was an attempt to uniformize the function names inside this class, especially because there was also a function named 'isEcludedTemplate'.
I will put back the 'checkIf...' part but see if it is possible to uniformize a bit and make it less "awful and painful" :)

Reply:
Done. Not perfect but we could refactor deeply later